### DISCUSSION TITLE
v1.0.0 | macOS native

### DISCUSSION THREAD

**Discussion by ThijmenDam** [ThijmenDam/BarTranslate#24] [Announcements]
My favorite release in a long time: BarTranslate is now a native macOS application! 😃

1. Bundle size from 80+ MB to 650 KB (!)
2. Resource usage is much lower
3. All the extra overhead from Electron is now gone. Bye!

Great, isn't it?

<hr /><em>This discussion was created from the release <a href='https://github.com/ThijmenDam/BarTranslate/releases/tag/v1.0.0'>v1.0.0 | macOS native</a>.</em>

**Comment by korkmazonur** [comment]
I tried all ver. but always “BarTranslate” can’t be opened because Apple cannot check it for malicious software.
I really want to use this app :)
I have Macbook air m2

**Reply by ThijmenDam** [reply]
Hi @korkmazonur! The message you are seeing is not directly related to BarTranslate, and can be circumvented easily.

Could you reproduce [steps described on the official Apple Support page](https://support.apple.com/guide/mac-help/apple-cant-check-app-for-malicious-software-mchleab3a043/mac), and let me know if that worked out for you? 

Glad to hear you'd like to use BarTranslate.

**Reply by korkmazonur** [reply]
Great It works thanks a lot
